### PR TITLE
Update tests with sirepo-bluesky release

### PR DESCRIPTION
**Pull request by jennmald**

### CODE REVIEW

File: deep_beamline_simulation/tests/test_ophyd_attempt.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+               'method': 'guest',
+               'needCompleteRegistration': False,
+               'paymentPlan': 'enterprise',
+               'roles': ['adm', 'enterprise', 'premium'],
+               'uid': '78msjrcn',
+               'upgradeToPlan': None,
+               'userName': None,
+               'visibleMethods': []},
+               'state': 'ok'}
+    assert str(expected['state']) == str(response_auth_guest_login.json()['state'])
```
Comment (jklynch) [inline deep_beamline_simulation/tests/test_ophyd_attempt.py:32]: Can tests like these be dictionary vs dictionary rather than str vs str?

File: deep_beamline_simulation/tests/test_ophyd_attempt.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+
+def test_simulationUpload():
+    session = requests.Session()
+    response_sim_list = session.post("http://localhost:8000/simulation-list", json={"simulationType": "srw"})
+    response_auth_guest_login = session.post("http://localhost:8000/auth-guest-login/srw")
+    files = {"file": open("../../example-2.zip", "rb"), "folder": (None, "/foo")}
```
Comment (jklynch) [inline deep_beamline_simulation/tests/test_ophyd_attempt.py:39]: use deep_beamline_simulation.__path__ to figure out the absolute path to example-2.zip

Comment (jklynch) [reply deep_beamline_simulation/tests/test_ophyd_attempt.py:39]: that reminds me "example-2.zip" needs a more informative name

File: tensorfile.py [hunk 3]
```diff
@@ -10,8 +9,8 @@
 file1.close()
 
 accuracy_list = []
-file_name2 = 'accuracy.txt'
-file2 = open(file_name2, 'r')
+file_name2 = "accuracy.txt"
+file2 = open(file_name2, "r")
```
Comment (jklynch) [inline tensorfile.py:13]: I recommend using the `while` statement whenever possible for opening and closing files to save yourself the trouble of 1) remembering to close the file when nothing goes wrong (which you have done) and 2) remembering to close the file when something goes wrong.

**Review by jklynch — CHANGES_REQUESTED** [review]
did accuracy.txt and loss.txt get deleted?

**Comment by jennmald** [comment]
> did accuracy.txt and loss.txt get deleted?

I updated them when making the presentation and had yet to push the changes since I ran it so many times to get graphs. I updated them and put them into gitignore since I didn't want to update them each time I ran the tensor file.

File: tensorfile.py (hunk 3 above)
Comment (mrakitin) [reply tensorfile.py:14]: `while` -> `with`?

Comment (jklynch) [reply tensorfile.py:14]: Yes, my mistake. I meant ``with``.

File: tensorfile.py [hunk 4]
```diff
@@ -1,20 +1,16 @@
-
 import numpy as np
 from torch.utils.tensorboard import SummaryWriter
 
 loss_list = []
-file_name1 = "loss.txt"
-file1 = open(file_name1, "r")
-for line in file1:
+with open("loss.txt", 'r') as file1:
+    line = file1.readline()
```
Comment (mrakitin) [inline tensorfile.py:6]: Did you intend to read only one first line? Or did you want to read all lines? For the latter case, `file1.read()` or `file1.readlines()` can be used.

**Review by mrakitin — COMMENTED** [review]
@jennmald, I had some comments before this PR was merged. Please have a look at them as they may be useful for your work on tests (#29).

File: deep_beamline_simulation/tests/test_ophyd_attempt.py [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+        Path(deep_beamline_simulation.__path__[0]).parent / "sirepo_simulations"
+    )
+    simulation_zip_file_path = sirepo_simulations_dir / "sim_example.zip"
+    files = {
+        "file": open(str(simulation_zip_file_path), "rb"),
+        "folder": (None, "/foo"),
+    }
+    response_import_file = session.post(
+        "http://localhost:8000/import-file/srw", files=files
+    )
```
[the remark this answers is not in the document]
Comment (jennmald) [reply deep_beamline_simulation/tests/test_ophyd_attempt.py:98]: I decided to create a function to handle some of the repeated code. I will try to put all of these changes on the same PR (#29 )

File: deep_beamline_simulation/tests/test_ophyd_attempt.py [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+    uploaded_sim["report"] = "intensityReport"
+    # run simulation here
+    response_run_simulation = session.post(
+        "http://localhost:8000/run-simulation", json=uploaded_sim
+    )
+    for i in range(1000):
+        state = (response_run_simulation.json())["state"]
+        if state == "completed" or state == "error":
+            state = (response_run_simulation.json())["state"]
+        if state == "completed" or state == "error":
```
[the remark this answers is not in the document]
Comment (jennmald) [reply deep_beamline_simulation/tests/test_ophyd_attempt.py:113]: fixed!

File: deep_beamline_simulation/tests/test_ophyd_attempt.py [hunk 7]
```diff
@@ -0,0 +1,196 @@
...
+        },
+        "title": "On-Axis Spectrum from Filament Electron Beam",
+        "x_label": "Photon Energy [eV]",
+        "x_range": [100.0, 20000.0],
+        "x_units": "eV",
+        "y_label": "Intensity (ph/s/.1%bw/mm²)",
+        "y_units": "ph/s/.1%bw/mm^2",
+        "z_range": [210561.34375, 1.8392213897609216e16],
+    }
+    actual_summary_data = response_run_simulation.json()["summaryData"]
```
[the remark this answers is not in the document]
Comment (jennmald) [reply deep_beamline_simulation/tests/test_ophyd_attempt.py:134]: fixed this as well, I had the assert at first then was second guessing what I was testing in it. The values stay the same for the simulation.
